src/Core/Service/Subscriber/LicenseSyncSubscriber.php

```diff
@@ -102,33 +97,27 @@ public function serviceInstalled(AppInstalledEvent|AppUpdatedEvent $event): void
             return;
         }
 
-        try {
-            $serviceEntry = $this->serviceRegistryClient->get($app->getName());
-            $this->syncLicenseByService($serviceEntry, $app, $context);
-        } catch (\Throwable $e) {
-            $this->logger->warning('Could not sync license', ['exception' => $e->getMessage()]);
-        }
+        $this->syncLicenseByService(
+            $app,
+            $context,
+            $this->config->getString(self::CONFIG_STORE_LICENSE_KEY),
+            $this->config->getString(self::CONFIG_STORE_LICENSE_HOST)
+        );
     }
 
-    private function syncLicenseByService(ServiceRegistryEntry $serviceEntry, AppEntity $app, Context $context, ?string $licenseKey = null, ?string $licenseHost = null): void
+    private function syncLicenseByService(AppEntity $app, Context $context, string $licenseKey, string $licenseHost): void
     {
-        if ($serviceEntry->licenseSyncEndPoint === null) {
-            return;
-        }
-
-        if ($licenseKey === null) {
-            $licenseKey = $this->config->getString(self::CONFIG_STORE_LICENSE_KEY);
-        }
+        try {
+            $serviceEntry = $this->serviceRegistryClient->get($app->getName());
 
-        if ($licenseHost === null) {
-            $licenseHost = $this->config->getString(self::CONFIG_STORE_LICENSE_HOST);
-        }
+            if ($serviceEntry->licenseSyncEndPoint === null) {
+                return;
+            }
 
-        try {
             $client = $this->clientFactory->newAuthenticatedFor($serviceEntry, $app, $context);
 
             $client->syncLicense($licenseKey, $licenseHost);
-        } catch (ServiceException|AppUrlChangeDetectedException $e) {
+        } catch (\Throwable $e) {
             $this->logger->warning('Could not sync license', ['exception' => $e->getMessage()]);
```

Comment:
Hi @fschmtt 

Not related to your MR, but IMO, we should "Silently ignore these exceptions as per service behavior." by removing the logger here, WDYT?

<img width="1459" alt="image" src="https://github.com/user-attachments/assets/68189211-8804-424a-b4c5-1f49f39dc055" />

Reply:
IIRC, during the first service release, some external developers asked why logs related to our services appeared during their Shopware install/update process. For example, why were there POST requests to https://preview... they never know.

So I’m wondering 🤔 if we should silence these logs related to the service installation process, like we already did in ServiceLifecycle::install

Reply:
using `->warning` should be okay, in order not to pollute systems with errors on our side. (since practically, nothing could be done from the shop)